tools/updateHeaders.pl

```diff
@@ -147,12 +190,25 @@
 my $exitCode = $?>> 8;
 if($exitCode != 0) {
 	 print "Error occurred during dicomTar!  Exit code was $exitCode\n" if $verbose;
-	 exit 1;
+	 exit $NeuroDB::ExitCodes::PROGRAM_EXECUTION_FAILURE;
 }
 
-exit 0;
+exit $NeuroDB::ExitCodes::SUCCESS;
+
+=pod
+
+=head3 extract_tarchive($tarchive, $tempdir)
 
+Extracts the DICOM archive given as an argument in a temporary directory and
```

Comment:
..DICOM archive __passed__ as argument in a...

Reply:
done